app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -99,6 +99,24 @@
     </div>
 }
 
+@dateFieldSet(datePrefix: String, dateLabel: String) = {
+
+    @editSetRecordForm.error(datePrefix+"-date-day") match {
+        case Some(_) => {
```

Comment:
You could remove the braces for both `Some` and `None`, so use a single line for each.

Reply:
Without the braces, there's an error compiling template and doesn't recognise `case`

Reply:
I meant on line 105 - but it's OK, as you've followed my suggested approach.